changelog.d/20230403_154257_alina.tuholukova_include_incident_url.md

```diff
@@ -0,0 +1,41 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Removed
+
+- A bullet item for the Removed category.
+
+-->
+<!--
+### Added
+
+- A bullet item for the Added category.
+
+-->
+
+### Changed
+
+- multiscan endpoint includes now incident_url the secret of a policy break is known by dashboard.
```

Comment:
The changelog fragment must be about the API py-gitguardian exposes. In this case we added a new field to PolicyBreak so the fragment could be something like this:

```suggestion
- PolicyBreak now includes the URL of the policy break if the dashboard already knows about it.
```

But this means it should be a `### Added` entry.

The fact that you mention the `multiscan` endpoint makes me wondering about the `scan` endpoint. Does it return `incident_url` too?

Reply:
ok, thanks. I'll change. No.. it's just for the ggshield users and "hidden" from public api users